views/admin/layout/sidebar/crmDataManager.cfm

```diff
@@ -1,17 +1,17 @@
 <cfscript>
     subMenuItems = [];
-    if ( hasCmsPermission( "organisationmanager.read" ) ) {
+    if ( hasCmsPermission( permissionKey="crm.organisation.navigate" ) ) {
         subMenuItems.append( {
-              link  = event.buildAdminLink( linkTo="organisationManager" )
+              link  = event.buildAdminLink( objectName="organisation" )
             , title = translateResource( "cms:crmDataManager.organisations" )
-            , active = ReFindNoCase( "\.?organisationManager$", event.getCurrentHandler() )
+            , active = ReFindNoCase( "^admin\.datamanager", event.getCurrentEvent() ) && ( prc.objectName ?: "" ) == "organisation"
         } );
     }
-    if ( hasCmsPermission( "personmanager.read" ) ) {
+    if ( hasCmsPermission( permissionKey="crm.person.navigate" ) ) {
         subMenuItems.append( {
-              link  = event.buildAdminLink( linkTo="personManager" )
+              link  = event.buildAdminLink( objectName="person" )
             , title = translateResource( "cms:crmDataManager.persons" )
-            , active = ReFindNoCase( "\.?personManager$", event.getCurrentHandler() )
+            , active = ReFindNoCase( "^admin\.datamanager", event.getCurrentEvent() ) && ( prc.objectName ?: "" ) == "person"
```

Comment:
see above - does this work when viewing/adding/editing a person address / link / note?

Reply:
Fixed